packages/scene-composer/src/utils/entityModelUtils/SceneLifecycleModule.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { TwinMakerSceneMetadataModule } from '@iot-app-kit/source-iottwinmaker';
+
+import { setTwinMakerSceneMetadataModule } from '../../common/GlobalSettings';
+import { SCENE_ROOT_ENTITY_COMPONENT_NAME, LAYER_ROOT_ENTITY_ID } from '../../common/entityModelConstants';
+import { SceneCapabilities } from '../../common/sceneModelConstants';
+import { emptyScene } from '../emptyScene';
+import { KnownSceneProperty } from '../../interfaces';
+
+import { SceneLifecycleModule } from './SceneLifecycleModule';
+import { updateSceneEntityComponent } from './sceneComponent';
+
+describe('createDynamicScene', () => {
+  const createScene = jest.fn();
+  const getSceneInfo = jest.fn().mockResolvedValue({
+    workspaceId: 'workspace-id',
+    sceneId: 'dynamic-scene',
+    contentLocation: undefined,
+    arn: 'dynamic-scene-arn',
+    creationDateTime: new Date(2022, 2, 1),
+    updateDateTime: new Date(2022, 2, 1),
+    capabilities: ['DYNAMIC_SCENE'],
+    sceneMetadata: { sceneRootEntityId: 'example-dynamic-scene-entity-id' },
+  });
+  const updateSceneEntity = jest.fn().mockResolvedValue({
+    updateDateTime: new Date(2022, 2, 1),
+    state: 'ACTIVE',
+  });
+  const createSceneEntity = jest.fn().mockResolvedValue({
+    entityId: LAYER_ROOT_ENTITY_ID,
```

Comment:
this layer entityId should be something random instead of `LAYER_ROOT_ENTITY_ID`, so people won't think it needs to be `LAYER_ROOT_ENTITY_ID`

Reply:
Does this mean that I should revise [this line](https://github.com/awslabs/iot-app-kit/pull/2615/files#diff-5c10115eeda6622861fe6a93c185292831e245eba5bfa58bccfcc694a928d3ffR18) as well?

Reply:
yes. `LAYER_ROOT_ENTITY_ID` is the parentEntityId of the layer entity

Reply:
Updated!